components/ModalSheet.js

```diff
@@ -43,15 +49,6 @@ export default ({
         swipeDirection="down"
         style={styles.modal}
       >
-        <AntDesign.Button
```

Comment:
I think you can remove https://github.com/echo-labs-team/the-resonance-project/blob/7805fc428ae6ab2c3c0b57fac4df2680e35dc777/components/ModalSheet.js#L11